### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tuesdayMarch**
Clarification on Linux installation.

I can conglomerate Linux dependencies and streamline the install some more. 

Don't know if 

>  Look into `.travis.yml` file for clues.

 is in the "spirit of the game" or just lazy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,16 +31,19 @@ It should be possible to build everything without Qt Creator, but the guide assu
...
-2. `./make.sh`. Built project will be placed under `exec` directory. Server will be
+1. Install dependencies. Look into `.travis.yml` file for clues. 
+2. `cd` into installation directory.
+3. `git clone https://github.com/griefly/griefly.git`
+4. `cd griefly`
+4. `./make.sh`. Built project will be placed under `exec` directory. Server will be
    built in `gopath/src/griefly-server` directory.
 
-**Note:** It is supposed to perform build from an active git repository (`git describe --tags --abbrev=4` will be called during the process).  
+**Note:** It's supposed to be bult from an active git repository (`git describe --tags --abbrev=4` will be called during the process) -- if you recieve a `fatal: Not a git repository` error, fix with `git init`.
```
Comment (mechmind) [inline README.md]: `git init` does not fix the error, just suppress it. Network protocol relies on git version for compatibility so you will not be able to connect to clients and servers built from actual repository.

Comment (kremius) [inline README.md]: Yep, @mechmind is right. It should me more like "if you receive ... then that is the case".

File: README.md [hunk 2]
```diff
@@ -31,16 +31,19 @@ It should be possible to build everything without Qt Creator, but the guide assu
    Client executables will appear in the `exec` folder, and the server executable will appear in the `griefly-server` folder.  
    Client executables depend from various dlls, so it is needed to manually place them to the `exec` folder.
 
-**Note:** It is supposed to perform build from an active git repository (`git describe --tags --abbrev=4` will be called during the process).
+**Note:** It's supposed to be built from an active git repository (`git describe --tags --abbrev=4` will be called during the process).
```
Comment (kremius) [inline README.md]: I am not sure why you have changed "It is" to "It's". In other places in the `README.md` reductions are not used.
Maybe there are some reason to use it here?

File: README.md (hunk 1 above)
Comment (kremius) [inline README.md]: Also it should be "built" instead of "bult" probably.

**Comment by kremius** [comment]
Thanks for your efforts!
Full list of dependencies is written above, so it makes sense only make an exact list of commands - and it is impossible to make it generic for linux (because different distros have different packet managers).
So if you wish to expand the `.travis.yml` reference to something more meaningful you can do it for your distro and say something like "For example, the following commands will install all needed dependcies on Yourdistroname:".

File: README.md (hunk 1 above)
Comment (tuesdayMarch) [inline README.md]: After doing a quick search, seems so.

If `git clone` is used, it shouldn't be an issue.

File: README.md (hunk 2 above)
Comment (tuesdayMarch) [inline README.md]: My pedantic contractions. Just removing clumsiness in language.

**Comment by tuesdayMarch** [comment]
@kremius 

Sure thing.
I know some gits have distro-specific install commands written for the more commonly used distros. Probably in the line of thought that anyone who can figure out GNU and Parabola can figure out how to get all of the dependencies. 
[Example](https://github.com/SecUpwN/Spotify-AdKiller#dependencies)

File: README.md (hunk 2 above)
Comment (kremius) [inline README.md]: Ok, if it looks better for native speakers then it is fine. Just was curious.

File: README.md [hunk 3]
```diff
@@ -31,16 +31,19 @@ It should be possible to build everything without Qt Creator, but the guide assu
...
-2. `./make.sh`. Built project will be placed under `exec` directory. Server will be
+1. Install dependencies. Look into `.travis.yml` file for clues. 
+2. `cd` into installation directory.
+3. `git clone https://github.com/griefly/griefly.git`
+4. `cd griefly`
+4. `./make.sh`. Built project will be placed under `exec` directory. Server will be
    built in `gopath/src/griefly-server` directory.
 
-**Note:** It is supposed to perform build from an active git repository (`git describe --tags --abbrev=4` will be called during the process).  
+**Note:** It's supposed to be built from an active git repository (`git describe --tags --abbrev=4` will be called during the process) -- if you recieve a `fatal: Not a git repository` error, then `git init` will surpress the issue, but will interfere with network connections.
```
Comment (kremius) [inline README.md]: 1. Two spaces at the end of the line should not be removed - it is a part of the markdown language. Without it looks wrong (you can check it yourself).
2.  `git init` should not be mentioned at all. It is a bad way to hide the issue and does not give any profit. The error means only one thing - the project should be cloned from a git repo.

**Pull request merged**